recipes/self-managed-otlp-ingest/README.md

```diff
@@ -0,0 +1,150 @@
+# Using Self Managed OTLP ingest with OpenTelemetry Operator
+
+This recipe shows how to use self-managed OTLP ingest to export telemetry from an auto-instrumented application deployed on a GKE cluster.
+
+The example demonstrates a scenario where a user has an application deployed on a GKE cluster and now needs to instrument it and export the generated telemetry to Google Cloud.
+
+## Setting up the cluster with an application
+
+> [!TIP]
+> This section outlines the steps to create a cluster & deploy a sample application. Feel free to skip this section and move to [Instrumenting deployed applications](#instrumenting-deployed-applications) if you already have a sample app and cluster setup. \
+> Alternatively, you can run a convenience script that executes the steps in this section to give you a cluster with a running application. You can execute this script from the root of this recipe: `./setup-application.sh`. 
+
+### Creating a GKE cluster
+
+Run the following commands to create a GKE autopilot cluster. Alternatively, you can create a cluster using the Google Cloud Platform UI.
+
+```shell
+# Setup required environment variables, modify based on your preference
+export PROJECT_ID="<your-project-id>"
+export CLUSTER_NAME="opentelemetry-autoinstrument-cluster"
+export CLUSTER_REGION="us-central1"
+```
+
+The following `gcloud` command will begin the cluster creation process.
+```shell
+gcloud beta container --project "${PROJECT_ID}" clusters create-auto "${CLUSTER_NAME}" --region "${CLUSTER_REGION}" --release-channel "regular" --tier "standard" --enable-ip-access --no-enable-google-cloud-access --network "projects/${PROJECT_ID}/global/networks/default" --subnetwork "projects/${PROJECT_ID}/regions/${CLUSTER_REGION}/subnetworks/default" --cluster-ipv4-cidr "/17" --binauthz-evaluation-mode=DISABLED
+```
+
+Connect `kubectl` to the created cluster:
+```shell
+gcloud container clusters get-credentials ${CLUSTER_NAME} --region ${CLUSTER_REGION} --project ${PROJECT_ID}
```

Comment:
this isn't needed if you just created the cluster

Reply:
I will confirm this once, but in either case - I think it is good to have it here, especially since this seems like a simple command.

This way the user can also connect to some pre-existing cluster.